### PR TITLE
More on ARTEMIS-594: support HTTPS access to hawtio

### PR DESCRIPTION
**Pull request by gaohoward**
Remove the keystore.jks in distribution
  Add documentation

### CODE REVIEW

**Comment by clebertsuconic** [comment]
what about adding --ssl-file <file> --ssl-password as part of the create command, and make it do it automatically for users. We wouldn't provide the keystore but would let it automated for users.

@andytaylor  ^ ?

**Comment by andytaylor** [comment]
makes sense

**Comment by clebertsuconic** [comment]
don't remove the docs though.. just add the options on the CLI to create

**Comment by gaohoward** [comment]
ok will do it.

**Comment by gaohoward** [comment]
@andytaylor @clebertsuconic  I've added the cli options.

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java [hunk 1]
```diff
@@ -347,6 +361,27 @@ public String getClusterPassword() {
       return clusterPassword;
    }
 
+   public String getSslKeyPassword() {
+      if (sslKeyPassword == null) {
+         sslKeyPassword = inputPassword("--ssl-key-password", "Please enter the keystore password:", "password");
```
Comment (clebertsuconic) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java]: I wouldn't force any of this to use input.. this is an extra feature... can you remove the inputs you have added?

**Comment by clebertsuconic** [comment]
@gaohoward  nice one howard!

**Comment by gaohoward** [comment]
Thanks @clebertsuconic :)

**Pull request merged**